versioned_docs/version-1.21/getting-started/installation.md

```diff
@@ -0,0 +1,308 @@
+
+Rancher Desktop makes use of Traefik as the default ingress controller. Users may run into a `permission denied` error after deploying Rancher Desktop due to restricted port access on the Traefik ingress. Most Linux distributions (e.g. Ubuntu 20.04) do not allow non-root users to listen on TCP and UDP ports below `1024`. In order to allow Traefik to listen to privileged ports on the local host, please run the following command:
+
+```
+sudo sysctl -w net.ipv4.ip_unprivileged_port_start=80
+```
+
+After the command is run, all ports `80` and above will become unprivileged and Traefik will be able to successfully access said ports.
+
+To preserve this change across reboots as a custom kernel parameter setting, add the same command inside your `/etc/sysctl.conf` file.
+
+### Installation via .deb Package
+
+Add the Rancher Desktop repository and install Rancher Desktop with:
+
+```
+curl -s https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/Release.key | gpg --dearmor | sudo dd status=none of=/usr/share/keyrings/isv-rancher-stable-archive-keyring.gpg
+echo 'deb [signed-by=/usr/share/keyrings/isv-rancher-stable-archive-keyring.gpg] https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/ ./' | sudo dd status=none of=/etc/apt/sources.list.d/isv-rancher-stable.list
+sudo apt update
+sudo apt install rancher-desktop
+```
+
+
+### Uninstalling .deb Package
+
+You can remove the package, repository, and key with:
+
+```
+sudo apt remove --autoremove rancher-desktop
+sudo rm /etc/apt/sources.list.d/isv-rancher-stable.list
+sudo rm /usr/share/keyrings/isv-rancher-stable-archive-keyring.gpg
+sudo apt update
+```
+
+
+### Installing via .rpm Package
+
+#### openSUSE
+
+To add the repository and install on openSUSE:
+
+```
+sudo zypper addrepo https://download.opensuse.org/repositories/isv:/Rancher:/stable/rpm/isv:Rancher:stable.repo
+sudo zypper install rancher-desktop
+```
+
+#### Fedora
+
+To add the repository and install on Fedora:
+
+```bash
+sudo dnf config-manager addrepo --from-repofile=https://download.opensuse.org/repositories/isv:/Rancher:/stable/fedora/isv:Rancher:stable.repo
+sudo dnf install rancher-desktop
+```
+
+:::note
+
+The Fedora packages are not tested against RHEL and related distributions; please use the AppImage
+instead.
+
+:::
+
+### Uninstalling .rpm Package
+
+Ensure that Rancher Desktop has exited (if not, it should appear in the dock) and do:
+
+#### openSUSE
+
+```
+sudo zypper remove --clean-deps rancher-desktop
+sudo zypper removerepo isv_Rancher_stable
+```
+
+#### Fedora
+
+```
+sudo dnf remove rancher-desktop
+sudo rm '/etc/yum.repos.d/isv:Rancher:stable.repo'
+```
+
+### Installing via AppImage
+
+First, ensure that `pass` and `gpg` are installed. For example, on Rocky:
+
+```
+# Enabled EPEL following upstream documentation; this is elided here.
+dnf install pass gnupg2
+```
+
+You may download the AppImage [here]. In order to run it, simply make it executable
```

Comment:
This might be another missing hyperlink.

Reply:
No:

```markdown
You may download the AppImage [here]. In order to run it, simply make it executable
and execute it. For better integration with your desktop, you may use [AppImageLauncher].

[here]:
https://download.opensuse.org/repositories/isv:/Rancher:/stable/AppImage/rancher-desktop-latest-x86_64.AppImage
[AppImageLauncher]:
https://github.com/TheAssassin/AppImageLauncher
```

Same for the `[releases page]` at the top.